CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/panes/DataSourceTitledPane.java

```diff
@@ -347,6 +361,8 @@ public void validityChanged(boolean isEnabled) {
      * @param perPluginParamMap the parameter values.
      */
     public void setParameterValues(final Map<String, String> perPluginParamMap) {
+        pluginCount++;
```

Comment:
I'm not seeing any updated tests in the corresponding tests for these changes?

Reply:
Thanks. I will look at the DataSourceTitledPaneNGTest to include my change.